.travis.yml

```diff
@@ -27,7 +27,7 @@ env:
 matrix:
   include:
     - env: RUNTIME=3.6 TOOLKITS="pyqt pyqt5 pyside2 wx"
-    - env: RUNTIME=3.6 TOOLKITS="pyqt5 wx" TRAITS_REQUIRES="^=6.0"
```

Comment:
Same comment as in TraitsUI:

Could this be a separate PR please? It would be good if we see more obviously at what point we stopped testing against Traits 6.0. It needs to be prominent. Also if we are shuffling things around between pyface, envisage and apptools, whereas the next release of envisage and apptools should still support Traits 6.0, we may not be able to drop Traits 6.0 support on Pyface this soon.

Reply:
fixed. i also added `--force-reinstall` like you mentioned in the sister PR in traitsui.